src/style.css

```diff
@@ -2,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+  padding: 0 8px;
+}
+
+.form {
+  position: relative;
+  margin: 0 auto;
+}
+
+.form::before {
+  content: '';
+  position: absolute;
+  z-index: 1;
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.form_big::before {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+}
+
+.form_small::before {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+  transform: translateY(0%);
+}
+
+.input {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  margin-top: 20px;
+  font-family: Avenir-Book, Arial, sans-serif;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  outline: none;
+}
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you have `margin-top: 20px;` on `.input`, ensure that you're not adding a bottom margin elsewhere to avoid potential margin collapse.